sql/src/main/java/org/apache/druid/sql/calcite/schema/SystemSchema.java

```diff
@@ -586,7 +638,8 @@ public TableType getJdbcTableType()
           StringUtils.toLowerCase(discoveryDruidNode.getNodeRole().toString()),
           druidServerToUse.getTier(),
           currentSize,
-          druidServerToUse.getMaxSize()
+          druidServerToUse.getMaxSize(),
+          NullHandling.defaultLongValue()
```

Comment:
hmm. shouldn't this always be zero?  Data nodes don't have any leader.

Reply:
ideally it should always be null because they don't have the concept of leadership at all and then you can distinguish between things which are leader, are not leader, and which don't have leaders. `NullHandling.defaultLongValue()` is null when `druid.generic.useDefaultValueForNull=false`, but in default mode numbers cannot be null so it is 0 there.

Reply:
yeah. that works. 👍